testing/api/devices/device_1.json

```diff
@@ -0,0 +1,55 @@
+{
+  "status": "Valid",
```

Comment:
The status field should not be passed into the API. Only the backend will determine whether it is valid or not.